components/sections/call_to_action/variant_c.tsx

```diff
@@ -3,91 +3,105 @@ import WebriQForm from "components/webriq-form";
 import { thankYouPageLink } from "helper";
 
 import { CTAProps } from ".";
+import { Text } from "components/ui/Text";
+import { Button } from "components/ui/Button";
+import { Form } from "components/ui/Form/Form";
+import { Input } from "components/ui/Input";
+import { Container } from "components/layout/Container";
+import { Flex } from "components/layout/Flex/Flex";
+import { Heading } from "components/ui/Heading";
 
 function VariantC({ title, text, features, form }: CTAProps) {
   return (
-    <section className="bg-gray-50 py-20">
-      <div className="container mx-auto px-4">
-        <div className="-mx-4 flex flex-wrap items-center">
-          <div className="mb-8 w-full px-4 lg:mb-0 lg:w-1/2">
-            <h1 className="font-heading mb-4 text-4xl font-bold lg:text-5xl">
-              {title}
-            </h1>
-            <p className="max-w-lg leading-loose text-gray-700">{text}</p>
+    <section className="py-20 bg-gray-50">
+      <Container>
+        <Flex
+          align="center"
+          className="flex-col lg:flex-row"
+          gap={4}
+          justify="between"
+        >
+          <div className="w-full px-4 mb-8 lg:mb-0 lg:w-1/2">
+            <Heading className="mb-4">{title}</Heading>
```

Comment:
Just like the other elements, kindly update the Title and Text elements to be returned only when it is defined/there is a value.